src/encoded/tests/test_search.py

```diff
@@ -753,3 +888,115 @@ def test_format_facets_adds_pseudo_facet_for_extra_filters():
         ],
         'total': 42,
     }]
+
+
+def test_sort_query(sort_query_item):
+
```

Comment:
I asked Karthik to remove these extra spaces a while ago.  Please remove them from the top of all these tests functions.